Reference/Searching/Examine/examine-events.md

```diff
@@ -95,6 +104,29 @@ So at this point we have done something along the lines of:
 - Get the valueset for the ExternalIndex
 - For each field, add the content to a new field called `combinedField`
 
+Next is an optional step to highlight how you can access the Umbraco published cache during an indexing event. In this example, we will be adding the breadcrumb values to our combined field.
+
+```cs
+private string GetBreadcrumb(string id)
+{
+    if (int.TryParse(idString, out var id))
+    {
+        using(var scope = _scopeProvider.Create(autocomplete: true))
+        {
+            using (var umbracoContextReference = _umbracoContextFactory.EnsureUmbracoContext())
+            {
+                var nodeBeingIndexed = umbracoContextReference.UmbracoContext.Content.GetById(id);
+                return nodeBeingIndexed.Ancestor != null ?  
+                    string.Join(" ", nodeBeingIndexed.AncestorsOrSelf().Select(n => n.Name)) :
+                    string.Empty;
+            }
+        }
+    }
+}
+```
+
+This example is here to highlight the user of the Scope Provider. A Scope is required for many things in Umbraco, from controlling database transactions to how service level events are handled to how nucache manages it's snapshots (and more). Although Umbraco tries to ensure there is a Scope available, this is not always the case, especially when working in background threads / events. As such, it is important to wrap calls to Umbraco Services/Contexts in a Scope, as without this unusual errors can occur, including the disposal of objects still required. *For more information on this, please see the answer to this [forum question](https://our.umbraco.com/forum/using-umbraco-and-getting-started/102676-triggering-index-rebuild-via-hangfire-causes-objectdisposedexception-in-nucache)*
```

Comment:
```suggestion
:::note
This example is here to highlight the user of the Scope Provider. A Scope is required for many things in Umbraco, from controlling database transactions to how service-level events are handled to how nucache manages it's snapshots (and more). Although Umbraco tries to ensure there is a Scope available, this is not always the case, especially when working in background threads/events. As such, it is important to wrap calls to Umbraco Services/Contexts in a Scope, as without this unusual errors can occur, including the disposal of objects still required. *For more information on this, please see the answer to this [forum question](https://our.umbraco.com/forum/using-umbraco-and-getting-started/102676-triggering-index-rebuild-via-hangfire-causes-objectdisposedexception-in-nucache)*
:::
```

Reply:
Just thought we should wrap the explanation in a 'note' - I also think we should add a note to the Services + Helpers page or have a scope page generally to link to, to flag the information. But in the context of this PR the sooner we add the note to the docs the sooner we help people!